pipeline/load.py

```diff
@@ -0,0 +1,151 @@
+
+    with conn.cursor(cursor_factory=RealDictCursor) as cur:
+        query = """
+        SELECT title_id
+        FROM title
+        WHERE title_name = %s;
+        """
+
+        cur.execute(query, title_name)
+        return cur.fetchone()
+
+
+def check_judge_exists(conn: connection, judges: list) -> list[int]:
+    """ Returns the judge_id if the judge exists in the judge table. """
+
+    judge_ids = []
+    for judge in judges:
+        judge = parse_name(judge)
+        title_id = get_title_id(judge['title'])
+        with conn.cursor(cursor_factory=RealDictCursor) as cur:
+            query = """
+            SELECT id
+            FROM judge
+            WHERE title_id = %s
+            AND last_name = %s;
+            """
+            cur.execute(query, (title_id, judge.get('last_name')))
+            judge_ids.append(cur.fetchone()[0])
+
+    return [id for id in judge_ids if id is not None]
+
+
+def get_judgement_id(conn: connection, ruling: str) -> int:
+    """ Returns the judgement ID that matches the ruling. """
+
+    with conn.cursor(cursor_factory=RealDictCursor) as cur:
+        query = """
+        SELECT judgement_id
+        FROM judgement
+        WHERE judgement_favour = %s;
+        """
+
+        cur.execute(query, ruling)
+        result = cur.fetchone()
+    return result if result else "Not found"
+
+
+def get_court_id(conn: connection, court_name: str) -> int:
+    """ Returns the court ID for a given court name. """
+
+    with conn.cursor(cursor_factory=RealDictCursor) as cur:
+        query = """
+        SELECT court_id
+        FROM court
+        WHERE court_name = %s;
+        """
+
+        cur.execute(query, court_name)
+        return cur.fetchone()
+
+
+def insert_into_court(conn: connection, court: str) -> int:
+    """ Insert a a new row into the court table and return its id. """
+
+    with conn.cursor(cursor_factory=RealDictCursor) as cur:
+        query = """
+        INSERT INTO court (court_name)
+        VALUES (%s)
+        RETURNING id;
+        """  # may need to change id to court_id (find out after testing)
+        cur.execute(query, court)
+        conn.commit()
+
+    return cur.fetchone()
+
+
+def insert_into_judge_hearing(conn: connection, judge_ids: list, hearing_id: int) -> None:
+    """ Inserts records in the judge_hearing table. """
+
+    for judge_id in judge_ids:
+        with conn.cursor(cursor_factory=RealDictCursor) as cur:
+            query = """
+            INSERT INTO judge_hearing (judge_id, hearing_id)
+            VALUES (%s, %s);
+            """
+            cur.execute(query, (judge_id, hearing_id,))
+            conn.commit()
+
+
+def insert_into_hearing(conn: connection, hearing: dict, metadata: dict) -> None:
+    """ Inserts a new row in the hearing table. """
+    judge_ids = check_judge_exists(conn, metadata.get('judges'))
+    if judge_ids:
+        judgement_id = get_judgement_id(conn, hearing.get(('ruling')))
```

Comment:
Extra pair of brackets here.
```suggestion
        judgement_id = get_judgement_id(conn, hearing.get('ruling'))
```